setup.py

```diff
@@ -9,114 +9,109 @@
-        'Programming Language :: Python :: 3.8',
-        'Programming Language :: Python :: 3.9',
-        'Programming Language :: Python :: 3.10',
-        'Programming Language :: Python :: 3.11',
-        'Programming Language :: Python :: Implementation :: CPython',
-        'Programming Language :: Python :: Implementation :: PyPy',
-        'Topic:: Scientific / Engineering:: Physics',
+        "Development Status :: 3 - Alpha",
+        "Intended Audience :: Science/Research",
+        "Operating System :: Unix",
+        "Operating System :: POSIX",
+        "Operating System :: Microsoft :: Windows",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
+        "Programming Language :: Python :: Implementation :: CPython",
+        "Programming Language :: Python :: Implementation :: PyPy",
+        "Topic:: Scientific / Engineering:: Physics",
     ],
     project_urls={
-        'Changelog': 'https://github.com/QuTech-Delft/QMI/CHANGELOG.md',
-        'Issue Tracker': 'https://github.com/QuTech-Delft/QMI/issues',
+        "Changelog": "https://github.com/QuTech-Delft/QMI/CHANGELOG.md",
+        "Issue Tracker": "https://github.com/QuTech-Delft/QMI/issues",
     },
     keywords=[
         # eg: 'keyword1', 'keyword2', 'keyword3',
     ],
-    python_requires='>=3.8, <4',
+    python_requires=">=3.8, <4",
     install_requires=[
         # For generating an installable package and deploying
-        'setuptools',
-        'wheel',
-        'twine',
+        "setuptools",
+        "wheel",
+        "twine",
         # For scientific data processing and visualisation
-        'numpy',
-        'scipy',
-        'h5py>=3.7.0',
+        "numpy",
+        "scipy",
+        "h5py>=3.7.0",
         # For hardware interfacing
-        'pyserial',
-        'pyusb',
-        'python-vxi11',
+        "pyserial",
+        "pyusb",
+        "python-vxi11",
         # For miscellaneous functionality
-        'pytz',
-        'psutil',
-        'colorama',
-        'jsonschema',
+        "pytz",
+        "psutil",
+        "colorama",
+        "jsonschema",
         # Instrument AD2
-        'pydwf',
+        "pydwf",
     ],
     extras_require={
-        'dev': [
+        "dev": [
             # For generating an installable package and deploying
-            'setuptools',
-            'wheel',
-            'twine',
+            "setuptools",
+            "wheel",
+            "twine",
             # For static code checks
             "astroid==2.12.2",
             "coverage",
-            'pylint',
-            'mypy',
-            "radon",
+            "pylint",
+            "mypy",
             # version bump
-            'bump2version'
+            "bump2version",
```

Comment:
We could remove `bump2version` in the OSS QMI, as we don't use it here. The releases and versioning is done manually. We can put it back if we get a procedure for releasing using it.

Reply:
works for me!